data/Collection.php

```diff
@@ -104,7 +104,7 @@ abstract class Collection extends \lithium\util\Collection {
 	 * will be assigned here.
 	 *
 	 * @see lithium\data\Schema
-	 * @var lithium\data\Schema
+	 * @var \lithium\data\Schema
```

Comment:
Our thing was always not to do leading `\` since it's (AFAIK) always implied and just looks like noise. Is there a compelling reason to change direction on that? If so, we should probably just reformat the entire codebase in one shot, rather than leave it inconsistent.